3_visualize.R

```diff
@@ -65,7 +65,7 @@ p3_targets_list <- list(
     p3_well_observed_site_data_json,
     {
       filename = "3_visualize/out/well_observed_trn_val_test.geojson"
-      st_write(p3_well_observed_site_data, filename, append=FALSE)
+      st_write(p3_well_observed_site_data, filename, append=FALSE, delete_layer=TRUE)
```

Comment:
The geojson file may be writing OK, but `delete_layer = TRUE` still leaves me with these errors:
```
GeoJSON' layer not supported by driver `
Deleting layer `well_observed_trn_val_test' failed
```

After checking out the [documentation](https://r-spatial.github.io/sf/reference/st_write.html) for `st_write`, I wonder if instead we need to specify `delete_dsn`. I'm curious if the change below works for you (I also made the driver explicit and added a quiet message to suppress verbose output when running `tar_make`:

```suggestion
      st_write(p3_well_observed_site_data, filename, append = FALSE, delete_dsn = TRUE, driver = "GeoJSON", quiet = TRUE)
```

Reply:
That works great! Thanks for looking into this, @lekoenig